doc/user/content/get-started.md

```diff
@@ -73,237 +62,465 @@ Download and install the [PostgreSQL installer](https://www.postgresql.org/downl
+     Signed Memorabilia |          14843
+    (5 rows)
+
+    Time: 32.064 ms
+    ```
 
-That's it! You just created your first transformations on streaming data using common SQL patterns, and tried to break Materialize! We encourage you to continue exploring the sample Kafka source, or try one of the use-case specific quickstarts. If you’re done for the time being, remember to clean up the environment:
+    Now that the materialzed view's results are served from memory, latency is low again.
 
-```sql
-DROP SCHEMA qck CASCADE;
+## Step 5. Survive failures
+
+Earlier, when you created your clusters, you gave each cluster one [replica](/overview/key-concepts/#cluster-replicas), that is, one physical resource. For the `ingest_qck` cluster, that's the max number of replicas allowed, as clusters for sources can have only one replica. For the `compute_qck` cluster, however, you can increase the number of replicas for greater tolerance to replica failure.
+
+Each replica in a non-source cluster is a logical clone, doing the same computation and holding the same results in memory. This design provides Materialize with active replication, and so long as one replica is still reachable, the cluster continues making progress.
+
+Let's see this in action.
+
+1. Add a second replica to the `compute_qck` cluster:
+
+    ```sql
+    CREATE CLUSTER REPLICA compute_qck.r2 SIZE = '2xsmall';
+    ```
 
-DROP CLUSTER quickstart;
+1. Check the status of the new replica:
+
+    ```sql
+    SHOW CLUSTER REPLICAS WHERE CLUSTER = 'compute_qck';
+    ```
+    <p></p>
+
+    ```noftm
+       cluster   | replica |  size   | ready
+    -------------+---------+---------+-------
+     compute_qck | r1      | 2xsmall | t
+     compute_qck | r2      | 2xsmall | t
+    (2 rows)
+    ```
+
+1. Once the `r2` replica is ready (`ready=t`), drop the `r1` replica to simulate a failure:
+
+    ```sql
+    DROP CLUSTER REPLICA compute_qck.r1;
+    ```
+
+1. Query the indexed view that you created in `compute_qck` earlier:
+
+    ```sql
+    SELECT * FROM avg_bids;
+    ```
+    <p></p>
+
+    ```nofmt
+            item        |    average_bid
+    --------------------+--------------------
+     Custom Art         | 49.770776201263864
+     Gift Basket        |   49.8909070204407
+     City Bar Crawl     | 50.056635368698046
+     Best Pizza in Town |  50.50023551577956
+     Signed Memorabilia |  50.11854192264935
+    (5 rows)
+
+    Time: 23.537 ms
+    ```
 
--- Reset the session variables to their original values,
--- so you can keep exploring
-RESET search_path;
+    As you can see, the results are available despite the failure of one of the cluster's replicas.
 
-RESET cluster;
+## Step 6. Scale up or down
+
+In addition to using replicas to increase fault tolerance, you can add and remove replicas to scale resources up or down according to the needs of a cluster. For example, let's say the `2xsmall` replica in the `compute_qck` cluster is running low on memory.
```

Comment:
```suggestion
In addition to using replicas to increase fault tolerance, you can add and remove replicas to scale resources up or down according to the needs of a cluster. For example, if the `2xsmall` replica in the `compute_qck` cluster is running low on memory.
```

Reply:
This change makes the sentence a fragment. Going to leave it as-is for now.